backend-lambda/services/productService.spec.js

```diff
@@ -68,4 +69,211 @@ describe("Product Service", () => {
       ).rejects.toThrow("Limit and offset must be numbers");
     });
   });
+
+  describe("Create new product", () => {
+    it("should create a new product", async () => {
+      // Setup
+      const product = {
+        product_id: 7,
+        business_id: 1,
+        category_id: 2,
+        name: "Summer shorts",
+      };
+
+      querySpy.mockResolvedValueOnce({ rowCount: 1 });
+
+      const keys = Object.keys(product).join(", ");
+      const values = Object.values(product);
+      const placeholders = values.map((_, i) => `$${i + 1}`).join(", ");
+
+      // Run
+      const res = await productService.createProduct(mockPool, product);
+
+      // Assert
+      expect(querySpy).toHaveBeenCalledWith(
```

Comment:
So this isn't a great test because in L#85-87 you just copied the code out of the implementing method so you're basically testing that 1=1. Instead, hard-code the expected test results (i.e. no interpolation in L#94)

Reply:
hardcoded